### PR TITLE
add report member button to alert banner

### PR DESCRIPTION
**Pull request by karlkeefer**
#### Proposed Changes

* adds report member button into the blocked-user alert banner, as discussed in #2431
* modifies the pre-commit shell script to run in docker if the trustroots container is running (I don't have `prettier` installed on my host machine)

option-to-report-in-banner

### CODE REVIEW

**Review by simison — DISMISSED** [review]
Thanks for the PR!

Button change looks good and tests well.

Can you move the Docker changes to another PR so that we we can look into that closer separately? I wouldn't want to block the progress on small improvement just because it's in the same PR. :-) Thanks for understanding!

Tested also mobile, which is a bit funky but I don't think it matters much for this banner:

image

**Comment by karlkeefer** [comment]
force-pushed a commit without the husky hook change!

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,4 +1,10 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npx lint-staged
+DOCKER_RUNNING=$(docker inspect --format="{{.State.Running}}" trustroots 2> /dev/null)
```
[the remark this answers is not in the document]
Comment (karlkeefer) [reply .husky/pre-commit]: Yeah... I'm sure there is a better approach. I was just trying to unblock my own commits and didn't really think it through 😓

**Review by simison — APPROVED** [review]
Thanks!

**Pull request merged**